packages/palette/src/elements/Button/Button.tsx

```diff
@@ -183,14 +184,34 @@ export class ButtonBase extends Component<ButtonBaseProps & SansProps> {
       >
         {loading && <Spinner size={this.props.buttonSize} />}
 
-        <Sans pt="1px" weight={weight || "medium"} color={color} size={size}>
+        <VisibleText
+          pt="1px"
+          weight={weight || "medium"}
+          color={color}
+          size={size}
+        >
           {children}
-        </Sans>
+        </VisibleText>
+        <HiddenText pt="1px" weight={weight || "medium"} size={size}>
```

Comment:
Might be worth adding `role="presentation"` here in case we care about accessibility one day 🙃 Don't want screen readers reading both labels out 😬

Reply:
Good idea, I didn't add because my understanding of screen readers is that they won't read anything visibly hidden but better to be clear anyway.

Reply:
Added this.